rig/machine_control/consts.py

```diff
@@ -304,3 +304,12 @@ class BMPInfoType(enum.IntEnum):
     can_status = 2  # Status of all CAN devices on the bus
     adc = 3  # ADC (e.g. voltage + temperature)
     ip_addr = 4  # IP Address
+
+
+# Dictionary of (address % 4, n_bytes % 4) to data type
+address_length_dtype = {
```

Comment:
I'm still not convinced I understand why this has to be a dictionary rather than just including the logic in the code (or a function). Is it measurably faster? It implies rather more magic than it implements, that's all... (e.g. as a function: https://github.com/project-rig/rig-scp/blob/master/lib/rs__scp.c#L36-L45)